include/flamegpu/gpu/CUDASimulation.h

```diff
@@ -125,6 +126,11 @@ class CUDASimulation : public Simulation {
      * @return False if an exit condition was triggered
      */
     void exitFunctions() override;
+    /**
+     * The simulation will sort agents periodically.
+     * @param n Agents will be sorted every n steps. 0 indicates no sorting
+     */
+    void setSortAgentsEveryNSteps(const unsigned int n);
```

Comment:
Is there a reason this isn't inside the `ModelDescription` hierarchy on a per-agent basis?